airflow/utils/process_utils.py

```diff
@@ -155,11 +162,13 @@ def execute_interactive(cmd: List[str], **kwargs):
     """
     log.info("Executing cmd: %s", " ".join(shlex.quote(c) for c in cmd))
 
-    old_tty = termios.tcgetattr(sys.stdin)
-    tty.setraw(sys.stdin.fileno())
+    if not IS_WINDOWS:
+        old_tty = termios.tcgetattr(sys.stdin)
+        tty.setraw(sys.stdin.fileno())
+
+        # open pseudo-terminal to interact with subprocess
+        master_fd, slave_fd = pty.openpty()
```

Comment:
I don’t think the function works after the patch (at least `slave_fd` would become undefined). Since this function is only used to run database commands, maybe we should mark this function as POSIX-only, and implement a separate function for Windows. (Would it be enough to simply brige stdin/out/err to the subprocess?)

Reply:
True, this is why the patch only works if this is being used as a Dask-Worker. I can add a check and raise a NotImplemented exception if the function is used on Windows. Would this be enough for the time being or can you tell me how to implement the Windows counterpart?

Reply:
I think this method is needlessly complicated and can be simplified to work same on both win/linux. 

We are not interested in the command output here. Simple `os.system(cmd)` should do the job on both Windows and Linux. We just need to change the cmd to be string and format it appropriately using `"` around filenames (sqlite and mysql) and find where the binaries are with `shutil.which`

I tested on linux that this works (for linux but shutil.which is cross-platform and should work on windows too).

```
python -c "import os,shutil; os.system(shutil.which('sqlite3'))"
```

Reply:
IIRC `os.system()` has issues on Windows waiting for the underlying command to finish (the process is executed asynchronously or something like that). Django’s `dbshell` (which is basically the same thing) simply uses `subprocess.run()` so we probably should do the same.

https://github.com/django/django/blob/225d96533a8e05debd402a2bfe566487cc27d95f/django/db/backends/base/client.py#L22